signac/__main__.py

```diff
@@ -1045,10 +1045,13 @@ def jobs():
                     try:
                         readline.read_history_file(fn_hist)
                         readline.set_history_length(1000)
-                    except (IOError, OSError) as error:
-                        if error.errno != errno.ENOENT:
-                            raise
-                    atexit.register(readline.write_history_file, fn_hist)
+                        with open(fn_hist, 'w') as f: pass
+                        atexit.register(readline.write_history_file, fn_hist)
+                    except FileNotFoundError:
+                        atexit.register(readline.write_history_file, fn_hist)
+                    except PermissionError:
+                        print("Warning: " +os.path.relpath(fn_hist) +" does not have read/write permission. "
```

Comment:
Try something like:
```suggestion
                        print(("Warning: {} does not have read/write permission. "
                               "The history of this shell will not be saved.").format(os.path.relpath(fn_hist)))
```
You'll probably need to edit the white space / etc. Just use `.format()`.